spec/mail/fields/common/common_field_spec.rb

```diff
@@ -16,15 +17,15 @@
     it "should return '' on to_s if there is no value" do
       expect(Mail::SubjectField.new(nil).to_s).to eq ''
     end
-    
+
     it "should leave ascii alone" do
       field = Mail::SubjectField.new("This is a test")
       expect(field.encoded).to eq "Subject: This is a test\r\n"
       expect(field.decoded).to eq "This is a test"
     end
-    
+
     it "should encode a utf-8 string as utf-8 quoted printable" do
-      value = "かきくけこ"
+      value = String.new("かきくけこ")
```

Comment:
Changing in another PR works too.

Magic comments were added in 1.9.0: https://github.com/ruby/ruby/commit/b1e97c9070e61ce4e7f1be0a02cef9ef302bba8f